src/cmd/prune.ts

```diff
@@ -0,0 +1,9 @@
+import prune from '../api/prune'
+import {PublicInstallationOptions} from '../api/install'
+
+export default (input: string[], opts: PublicInstallationOptions) => {
+  if (input && input.length) {
+    return prune(input, opts)
+  }
+  return prune(opts)
```

Comment:
But opts seems more natural as a last parameter. Maybe lets create two separate functions? Or lets move input to a property in opts?

If we decide to change the order of args, I guess we have to move opts to the beginning in all the API functions for consistency